src/components/Policy/Table/__tests__/LastTriggeredCell.tests.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import renderer from 'react-test-renderer';
+import { render } from '@testing-library/react';
+
+import { LastTriggeredCell } from '../LastTriggeredCell';
+
+describe('src/components/Policy/Table/LastTriggeredCell', () => {
+
+    const now = new Date(2020, 0, 1).getTime();
+
+    beforeAll(() => {
+        jest.spyOn(Date, 'now').mockImplementation(() => now);
+    });
+
+    it('renders correctly', () => {
+        const tree = renderer
+        .create(<LastTriggeredCell isEnabled={ true } lastEvaluation={ new Date(2020, 4, 10) }/>)
+        .toJSON();
+        expect(tree).toMatchSnapshot();
```

Comment:
Isn't this super brittle?

Reply:
Actually, yes, the whole point (IIUC) of it is to ensure you are aware that whatever you did made changes to the DOM.

I have no strong opinion but i can see the disadvantages of this (people will just ignore and update snapshots to fix the test error).
For the time being I'll remove the `toMatchSnapshot` and just expect that it does not throw any error. 
Until I find any better advantage than knowing that our changes updated the DOM it will remain snapshot-less.